### ISSUE TITLE
Uncaught TypeError: Cannot read property 'pane' of null

### ISSUE THREAD

**Issue by jiapei100** [atom/tabs#332]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.9.0-dev-7253514
**System**: Ubuntu 16.04
**Thrown From**: [tabs](https://github.com/atom/tabs) package, v0.97.0

### Stack Trace

Uncaught TypeError: Cannot read property 'pane' of null

```
At /usr/local/share/atom/resources/app.asar/node_modules/tabs/lib/layout.js:80

TypeError: Cannot read property 'pane' of null
    at Object.module.exports.paneForTab (/usr/local/share/atom/resources/app.asar/node_modules/tabs/lib/layout.js:80:31)
    at Object.module.exports.isOnlyTabInPane (/usr/local/share/atom/resources/app.asar/node_modules/tabs/lib/layout.js:86:60)
    at Object.module.exports.drag (/usr/local/share/atom/resources/app.asar/node_modules/tabs/lib/layout.js:23:25)
    at tabs-tab.ondrag (/usr/local/share/atom/resources/app.asar/node_modules/tabs/lib/tab-view.js:51:23)
```

### Commands

```
     -6:39.8.0 core:move-left (atom-text-editor.editor.is-focused)
     -6:38.9.0 core:move-down (atom-text-editor.editor.is-focused)
     -6:38 core:backspace (atom-text-editor.editor.is-focused)
     -6:37.5.0 core:delete (atom-text-editor.editor.is-focused)
     -6:37.1.0 core:save (atom-text-editor.editor.is-focused)
```

### Config

```json
{}
```

### Installed Packages

```coffee
# User
atom-alignment, v0.12.1 (active)
atom-typescript, v8.10.2 (active)
autocomplete, v0.47.0 (inactive)
autocomplete-html-entities, v0.1.0 (active)
diff-pane, v0.1.0 (inactive)
javascript-snippets, v1.2.1 (active)
language-html5-subtlegradient, v0.1.0 (active)
linter, v1.11.4 (active)
meld-diff, v0.2.0 (active)
split-diff, v1.0.0 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.51.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.43.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.3.1 (active)
one-light-ui, v1.3.1 (inactive)
one-dark-syntax, v1.2.0 (active)
one-light-syntax, v1.2.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.2 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.0 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.0 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.38.1 (active)
fuzzy-finder, v1.2.0 (active)
git-diff, v1.0.1 (active)
find-and-replace, v0.198.0 (inactive)
go-to-line, v0.30.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.57.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (active)
notifications, v0.64.1 (active)
open-on-github, v1.1.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.237.0 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.2.6 (active)
styleguide, v0.45.2 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.97.0 (active)
timecop, v0.33.1 (active)
tree-view, v0.208.0 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.32.2 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.0 (active)
language-clojure, v0.20.0 (active)
language-coffee-script, v0.47.0 (active)
language-csharp, v0.12.1 (active)
language-css, v0.36.1 (active)
language-gfm, v0.86.0 (active)
language-git, v0.13.0 (active)
language-go, v0.42.0 (active)
language-html, v0.44.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.19.0 (active)
language-javascript, v0.110.0 (active)
language-json, v0.18.0 (active)
language-less, v0.29.3 (active)
language-make, v0.22.0 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.0 (active)
language-property-list, v0.8.0 (active)
language-python, v0.44.0 (active)
language-ruby, v0.68.5 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.52.0 (active)
language-shellscript, v0.22.2 (active)
language-source, v0.9.0 (active)
language-sql, v0.21.1 (active)
language-text, v0.7.1 (active)
language-todo, v0.27.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.6 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by dcrystalj** [comment]
take a look at ubuntu, I am using 19.0 beta. Pane moving on hidpi monitor is still big issue in Atom

![Image pane crashing](http://shrani.si/f/2G/Cl/1MGOVOhy/anim.gif)

**Comment by smlombardi** [comment]
This happened to me when I dragged out a tab without even meaning to.  Mac 10.11.5.

**Comment by rochford77** [comment]
I also had this issue. It also was happening in nuclide (different error, was still complaining about mercurial). 

first of all, I got just reinstalled my os (OS X El Captain) last night and never had this problem before. I downloaded Mercurial from brew, cloned my repos set-up Atom, only to run into this problem. Basically any HG related extension was causing Atom to freak out. Here is how i fixed it:

Uninstalled/Reinstalled Atom (with Atom-HG), didn't fix.

uninstalled Atom, deleted .atom (hidden) folder from home folder, rebooted, reinstalled Atom (and atom-hg), didn't fix.

uninstalled Atom, deleted .atom, uninstalled Mercurial (brew), deleted my repos from my machine, rebooted, installed Mercurial (not using brew), installed Atom (and atom-hg), then pulled my repos in and its all golden. 

hope this helps.

**Comment by brianblakely** [comment]
Just ran into this.

[Enter steps to reproduce below:]

1. Open multiple tabs
2. Split up, which creates a blank pane above tab row
3. Drag tab into upper pane
4. Drag tab back down into lower tab row

**Atom Version**: 1.9.1
**System**: Microsoft Windows 7 Enterprise 
**Thrown From**: [tabs](https://github.com/atom/tabs) package, v0.98.0


### Stack Trace

Uncaught TypeError: Cannot read property 'pane' of null

```
At C:\Users\username\AppData\Local\atom\app-1.9.1\resources\app.asar\node_modules\tabs\lib\layout.js:80

TypeError: Cannot read property 'pane' of null
    at Object.module.exports.paneForTab (C:\Users\username\AppData\Local\atom\app-1.9.1\resources\app.asar\node_modules\tabs\lib\layout.js:80:31)
    at Object.module.exports.end (C:\Users\username\AppData\Local\atom\app-1.9.1\resources\app.asar\node_modules\tabs\lib\layout.js:55:23)
    at tabs-tab.ondragend (C:\Users\username\AppData\Local\atom\app-1.9.1\resources\app.asar\node_modules\tabs\lib\tab-view.js:54:23)
```

### Commands

```
     -1:37.9.0 about:view-release-notes (atom-workspace.workspace.scrollbars-visible-always.seti-compact.seti-icons.seti-roboto.seti-animate.theme-atom-dark-syntax.theme-seti-ui)
```

### Config

```json
{
  "core": {
    "audioBeep": false,
    "disabledPackages": [
      "terminal-status"
    ],
    "themes": [
      "seti-ui",
      "atom-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-jade, v0.3.0 (active)
atom-material-syntax, v0.4.6 (inactive)
atom-material-ui, v1.3.3 (inactive)
atom-ternjs, v0.14.2 (active)
emmet, v2.4.3 (active)
language-babel, vundefined (active)
language-shadercode, v2.0.0 (active)
linter, v1.11.14 (active)
linter-eslint, v7.2.4 (active)
minimap, v4.24.7 (active)
seti-syntax, v1.0.1 (inactive)
seti-ui, v1.3.2 (active)
terminal-status, v1.6.8 (inactive)
atom-dark-syntax, v0.27.0 (active)
atom-dark-ui, v0.51.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.43.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.3.2 (inactive)
one-light-ui, v1.3.2 (inactive)
one-dark-syntax, v1.2.0 (inactive)
one-light-syntax, v1.2.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.2 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.0 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.0 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.38.1 (active)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
find-and-replace, v0.198.0 (inactive)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (active)
notifications, v0.64.1 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.238.0 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.4.0 (active)
styleguide, v0.46.0 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.98.0 (active)
timecop, v0.33.1 (active)
tree-view, v0.208.0 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.32.2 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.0 (active)
language-csharp, v0.12.1 (active)
language-css, v0.36.2 (active)
language-gfm, v0.86.0 (active)
language-git, v0.13.0 (active)
language-go, v0.42.0 (active)
language-html, v0.44.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.22.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.0 (active)
language-less, v0.29.3 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.0 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.68.5 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.52.0 (active)
language-shellscript, v0.22.3 (active)
language-source, v0.9.0 (active)
language-sql, v0.21.1 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.8 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by rochford77** [comment]
See here 

https://github.com/atom/status-bar/issues/161

**Comment by 50Wliu** [comment]
@rochford77 That has nothing to do with this issue.

**Comment by andela-cmutembei** [comment]
I was able to reproduce it on Mac OS X 10.11.6, Atom v1.9.4

**Comment by e5an** [comment]
I am also getting this error on OSX 10.11.5 / Atom 1.9.4 using Tabasco to drag panes around.

**Comment by amaslenn** [comment]
Reproduced with Atom 1.9.5 on Ubuntu 16.04.

**Comment by nathansobo** [comment]
This will be fixed in 1.9.8.
